src/test/java/com/Saurabh/GitHubAiAPI/StringOperationsTest.java

```diff
@@ -0,0 +1,101 @@
+package com.Saurabh.GitHubAiAPI;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class StringOperationsTest {
+
+
+    @BeforeAll
+    public static void setUp() {
+        // TO DO
+    }
+
+    // To DO
+    // test to check if the count of vwels is correct
+    @Test
+    public void testCountVowels_noVowels() {
+
+        assertEquals(0, StringOperations.countVowels("bcdfg"));
+    }
+
+
+    @Test
+    public void testCountVowels_emptyString() {
+        assertEquals(0, StringOperations.countVowels(""));
+    }
+
+    @Test
+    public void testCountVowels_onlyVowels() {
+        assertEquals(10, StringOperations.countVowels("aeiouAEIOU"));
+    }
+
+    @Test
+    public void testReverseString() {
+        assertEquals("olleh", StringOperations.reverseString("hello"));
+    }
```

Comment:
The test method testReplaceCharacters_withVowels could be more descriptive. Consider renaming it to indicate the expected outcome.